dbt/adapters/databricks/impl.py

```diff
@@ -429,10 +429,20 @@ def parse_columns_from_information(  # type: ignore[override]
         return columns
 
     def get_catalog(
-        self, manifest: Manifest, selected_nodes: Optional[Set] = None
+        self, manifest: Manifest, selected_nodes: Optional[Set[Any]] = None
```

Comment:
I believe we added `selected_nodes` as an argument and then removed it again, both in `1.7.0rc1`. However we forgot to remove it from `dbt-spark`. I'm verifying this now.

Reply:
We accidentally left this in for `dbt-spark`, so it actually makes sense to leave it here for now. Please disregard.